CDCgov/prime-reportstream
# Add fields to receiver settings

This PR adds dropdown fields and values for `timeZone` and `dateTimeFormat` Receiver settings properties.

Test Steps:
1. Ensure the fields populate with the right values on load
2. Ensure the fields apply changes to the proper fields

## Changes
Data from load
<img width="210" alt="Screen Shot 2023-01-25 at 11 13 41 AM" src="https://user-images.githubusercontent.com/4022304/214672566-6d78a4e4-0c3c-4765-a6e7-8630937283e2.png">

Data after editing with dropdown fields
<img width="812" alt="Screen Shot 2023-01-25 at 11 13 54 AM" src="https://user-images.githubusercontent.com/4022304/214672567-ddb91dc1-1612-4feb-ba7a-4ed6e63cf557.png">


## Checklist

### Testing

### Process

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
